src/utility/STM32Cube_FW/tl_mbox.c

```diff
@@ -532,181 +531,7 @@ __WEAK void TL_TRACES_EvtReceived( TL_EvtPacket_t * hcievt )
-        {
-          TL_HCI_EVT_DBG_MSG(" payload:");
-          TL_HCI_EVT_DBG_BUF(((TL_AsynchEvt_t*)(p_evt_packet->evtserial.evt.payload))->payload, p_evt_packet->evtserial.evt.plen-2, "");
-        }
-      }
-
-      TL_HCI_EVT_DBG_MSG("\r\n");
-
-      TL_HCI_EVT_DBG_RAW(&p_evt_packet->evtserial, p_evt_packet->evtserial.evt.plen+TL_EVT_HDR_SIZE);
-      break;
-
-    case TL_MB_SYS_CMD:
-      p_cmd_packet = (TL_CmdPacket_t*)buffer;
-
-      TL_SHCI_CMD_DBG_MSG("sys cmd: 0x%04X", p_cmd_packet->cmdserial.cmd.cmdcode);
-
-      if(p_cmd_packet->cmdserial.cmd.plen != 0)
-      {
-        TL_SHCI_CMD_DBG_MSG(" payload:");
-        TL_SHCI_CMD_DBG_BUF(p_cmd_packet->cmdserial.cmd.payload, p_cmd_packet->cmdserial.cmd.plen, "");
-      }
-      TL_SHCI_CMD_DBG_MSG("\r\n");
-
-      TL_SHCI_CMD_DBG_RAW(&p_cmd_packet->cmdserial, p_cmd_packet->cmdserial.cmd.plen+TL_CMD_HDR_SIZE);
-      break;
-
-    case TL_MB_SYS_CMD_RSP:
-      p_evt_packet = (TL_EvtPacket_t*)buffer;
-      switch(p_evt_packet->evtserial.evt.evtcode)
-      {
-        case TL_BLEEVT_CC_OPCODE:
-          TL_SHCI_CMD_DBG_MSG("sys rsp: 0x%02X", p_evt_packet->evtserial.evt.evtcode);
-          TL_SHCI_CMD_DBG_MSG(" cmd opcode: 0x%02X", ((TL_CcEvt_t*)(p_evt_packet->evtserial.evt.payload))->cmdcode);
-          TL_SHCI_CMD_DBG_MSG(" status: 0x%02X", ((TL_CcEvt_t*)(p_evt_packet->evtserial.evt.payload))->payload[0]);
-          if((p_evt_packet->evtserial.evt.plen-4) != 0)
-          {
-            TL_SHCI_CMD_DBG_MSG(" payload:");
-            TL_SHCI_CMD_DBG_BUF(&((TL_CcEvt_t*)(p_evt_packet->evtserial.evt.payload))->payload[1], p_evt_packet->evtserial.evt.plen-4, "");
-          }
-          break;
-
-        default:
-          TL_SHCI_CMD_DBG_MSG("unknown sys rsp received: %02X", p_evt_packet->evtserial.evt.evtcode);
-          break;
-      }
-
-      TL_SHCI_CMD_DBG_MSG("\r\n");
-
-      TL_SHCI_CMD_DBG_RAW(&p_evt_packet->evtserial, p_evt_packet->evtserial.evt.plen+TL_EVT_HDR_SIZE);
-      break;
-
-    case  TL_MB_SYS_ASYNCH_EVT:
-      p_evt_packet = (TL_EvtPacket_t*)buffer;
-      if(p_evt_packet->evtserial.evt.evtcode != TL_BLEEVT_VS_OPCODE)
-      {
-        TL_SHCI_EVT_DBG_MSG("unknown sys evt received: %02X", p_evt_packet->evtserial.evt.evtcode);
-      }
-      else
-      {
-        TL_SHCI_EVT_DBG_MSG("sys evt: 0x%02X", p_evt_packet->evtserial.evt.evtcode);
-        TL_SHCI_EVT_DBG_MSG(" subevtcode: 0x%04X", ((TL_AsynchEvt_t*)(p_evt_packet->evtserial.evt.payload))->subevtcode);
-        if((p_evt_packet->evtserial.evt.plen-2) != 0)
-        {
-          TL_SHCI_EVT_DBG_MSG(" payload:");
-          TL_SHCI_EVT_DBG_BUF(((TL_AsynchEvt_t*)(p_evt_packet->evtserial.evt.payload))->payload, p_evt_packet->evtserial.evt.plen-2, "");
-        }
-      }
-
-      TL_SHCI_EVT_DBG_MSG("\r\n");
-
-      TL_SHCI_EVT_DBG_RAW(&p_evt_packet->evtserial, p_evt_packet->evtserial.evt.plen+TL_EVT_HDR_SIZE);
-      break;
-
-    default:
-      break;
-  }
-
+  /* Function stubbed */
```

Comment:
OK function stubbed but generate warning about unused parameters:
```
STM32duinoBLE\src\utility\STM32Cube_FW\tl_mbox.c: In function 'OutputDbgTrace':
STM32duinoBLE\src\utility\STM32Cube_FW\tl_mbox.c:532:47: warning: unused parameter 'packet_type' [-Wunused-parameter]
  532 | static void OutputDbgTrace(TL_MB_PacketType_t packet_type, uint8_t* buffer)
      |                            ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
STM32duinoBLE\src\utility\STM32Cube_FW\tl_mbox.c:532:69: warning: unused parameter 'buffer' [-Wunused-parameter]
  532 | static void OutputDbgTrace(TL_MB_PacketType_t packet_type, uint8_t* buffer)
      |    
```                                                        ~~~~~~~~~^~~~~~

Note: think to update the linked patch 😉

Reply:
Thanks. Done